core/src/main/java/com/nukesz/game/BodyPart.java

```diff
@@ -3,24 +3,35 @@
 import com.badlogic.gdx.graphics.Texture;
 import com.badlogic.gdx.graphics.g2d.Batch;
 
-public class BodyPart {
-    public int x;
-    public int y;
-    private Texture texture;
+public class BodyPart extends Item {
+
 
     public BodyPart(Texture texture) {
-        this.texture = texture;
+        super(0, 0, texture);
+    }
+
+    public BodyPart(int x, int y) {
+        super(x, y, null);
     }
 
     public void updateBodyPosition(int x, int y) {
         this.x = x;
         this.y = y;
     }
 
+    public int getX() {
+        return x;
+    }
+
+    public int getY() {
```

Comment:
In that sense, then it should be car.adjustSpeedBy(10); or car.adjustSpeedBy(-10);  could work. 
But I accept and I will change it back as I didn't know this was the reason.